sirepo/template/myapp.py

```diff
@@ -25,6 +26,9 @@
 
 
 def get_data_file(run_dir, model, frame, options=None, **kwargs):
+    if 'suffix' in options and options.suffix == 'sr_long_analysis':
```

Comment:
options may be None so this will fail.

Reply:
Thanks, this should be on my #1435 branch which is still WIP. Reverting.